src/pageObject.js

```diff
@@ -1,16 +1,38 @@
 'use strict';
 
 class Header {
-
-};
+  get logo() {
+    return 'Conduit logo';
+  }
+}
 
 class PageObject {
-
-};
+  constructor(url, header = new Header()) {
+    this.header = header;
+    this.url = url;
+  }
+  clickOnLogo() {
+    return 'Click on the ' + this.header.logo;
+  }
+}
 
 class ArticlePage extends PageObject {
+  constructor(url, slug) {
+    super();
```

Comment:
The `super()` call in the `ArticlePage` constructor should pass the `url` parameter to the `PageObject` constructor. This ensures that the `url` property is correctly initialized in the parent class. Consider changing `super();` to `super(url);`.